tests/subsys/llext/simple/src/movwmovt_ext.c

```diff
@@ -12,20 +12,24 @@
 #include <stdint.h>
 #include <zephyr/llext/symbol.h>
 #include <zephyr/sys/printk.h>
+#include <zephyr/ztest_assert.h>
 
+static int test_var;
 
-static void test_func(void)
+static __used void test_func(void)
 {
 	printk("%s\n", __func__);
+	test_var = 1;
 }
 
 void test_entry(void)
 {
-	test_func();
+	test_var = 0;
 
 	printk("test movwmovt\n");
 	__asm volatile ("movw r0, #:lower16:test_func");
 	__asm volatile ("movt r0, #:upper16:test_func");
 	__asm volatile ("blx r0");
+	zassert_equal(test_var, 1, "mov.w and mov.t test failed");
```

Comment:
Nit:
```suggestion
	zassert_equal(test_var, 1, "movw and movt test failed");
```
(only the `.w` suffix exists, but it isn't available for the always-wide `movw`/`movt` instructions)

Reply:
Shoot... I saw that and I actually though I changed it. :man_facepalming: Sorry, next time I guess!